### PR TITLE
[BUG 1586874][UI] Profile shows n/a for scopes #1518

### PR DESCRIPTION
**Pull request by ANURADHAJHA99**
Bugzilla Bug: [XXXXX](https://bugzilla.mozilla.org/show_bug.cgi?id=XXXXX)

### CODE REVIEW

**Comment by ANURADHAJHA99** [comment]
@djmitche I have done the PR. Kindly please check. Thank you for the feedback.

File: ui/src/views/Profile/index.jsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ export default class Profile extends Component {
                         ))}
                       </List>
                     ) : (
-                      'n/a'
+                      'no scope'
```
Comment (djmitche) [inline ui/src/views/Profile/index.jsx]:
```suggestion
                      'no scopes'
```

**Comment by djmitche** [comment]
If you can commit the suggestion above, I can merge this PR :)

**Comment by ANURADHAJHA99** [comment]
@djmitche I have added the changes as requested. Thank you for the feedback.

**Comment by djmitche** [comment]
Thanks!

**Comment by helfi92** [comment]
@all-contributors please add @ANURADHAJHA99  for code

**Pull request merged**